TG-Netflix/src/components/Modal/Modal.style.js

```diff
@@ -72,11 +72,12 @@ export const VideoControlsContainer = styled.div`
   padding-left: 3rem;
   background: linear-gradient(0deg, #181818, transparent 50%);
 `;
-export const VideoTitle = styled.div`
+export const VideoTitle = styled.img`
```

Comment:
is dit een styled .img of toch een div?

Reply:
> is dit een styled .img of toch een div?

Ik heb het omgezet naar een `img` omdat de afbeelding niet zichtbaar was bij het doorgeven van een background-image via het style attribuut